cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java

```diff
@@ -189,6 +178,9 @@ protected List<HasMetadata> doReify(
                 .addToRequests("memory", new Quantity("256Mi"))
                 .addToLimits("cpu", new Quantity("500m"))
                 .addToLimits("memory", new Quantity("1Gi"))
+                .build())
+            .withLogging(new InlineLoggingBuilder()
+                .withLoggers(loggingMap)
```

Comment:
This seems doable. Would it be possible to use external logging configuration via a ConfigMap and update the ConfigMap in case of an enabled DEBUG mode?
We don't have to do this now, but maybe we can explore this solution in the future? Could you create a ticket for this?

Reply:
Issue MGDCTRS-1829